zipline/assets/_assets.pyx

```diff
@@ -240,7 +241,8 @@ cdef class Future(Asset):
                   object expiration_date=None,
                   object first_traded=None,
                   object exchange="",
-                  int contract_multiplier=1):
+                  int contract_multiplier=1,
+                  object fuzzy=""):
```

Comment:
Why does the fuzzy have to be a field of the Asset, rather than just having fuzzies at look-up in the AssetFinder?